Create privacy policy (#528)
icssc-projects/AntAlmanac: PRIVACY-POLICY.md

<h1 align=center> PRIVACY POLICY </h1>

> This privacy policy describes what, how, and why <em>AntAlmanac</em> and the technologies that it implements collect, store, and utilize the information that you or your device shares through your use of the website. We take reasonable precautions to protect your data, but you should NOT enter sensitive information into <em>AntAlmanac</em>. We may change this policy when necessary. By using <em>AntAlmanac</em>, you consent to this privacy statement.

Reviewer: ```suggestion
This privacy policy describes what, how, and why *AntAlmanac* and the technologies that it implements collect, store, and utilize the information that you or your device shares through your use of the website. We take reasonable precautions to protect your data, but you should NOT enter sensitive information into *AntAlmanac*. We may change this policy when necessary. By using *AntAlmanac*, you consent to this privacy statement.
```

> <h2> Information that we collect </h2>

Reviewer: ```suggestion
## Information that we collect
```

> <b> Non-tracking information: </b> 

Reviewer: ```suggestion
**Non-tracking information:**
```

<br></br>
<b> Study plans and usernames: </b> 
Your username(s) and the associated schedules and enrolled classes. This allows you to save your schedules and is NOT shared with third parties.
<br></br>
<b> Google Analytics usage data: </b> 
We use <em>Google Analytics</em> to track your usage, for example, which button you click on our your search queries. We do so to inform our prioritization and design of features. <em>Google Analytics</em> uses cookies and other tracking technologies to collect and store your information. Your usage information is anonymous and aggregated to us. HOWEVER, <em>Google Analytics</em> might collect your information individually or track you across websites. To block <em>Google Analytics</em>, use <em>Google</em>’s first-party <a href="https://support.google.com/analytics/answer/181881?hl=en">browser add-on</a>.
<br></br>
<em>

>   We do not collect any personally identifiable information. Your username should not contain your real name, and you should not enter sensitive information into AntAlmanac.

Reviewer: Since we had a "information we collect" section, would it make sense to have a "information we *don't* collect section"?

Reviewer 2: ~~There is no information we don't collect 😈~~
JK, this is a good idea.

<h1 align=center> PRIVACY POLICY </h1>

> This privacy policy describes what, how, and why *AntAlmanac* and the technologies that it implements collect, store, and utilize the information that you or your device shares through your use of the website. We take reasonable precautions to protect your data, but you should NOT enter sensitive information into *AntAlmanac*. We may change this policy when necessary. By using *AntAlmanac*, you consent to this privacy statement.

Reviewer: ```suggestion
This privacy policy describes *AntAlmanac*'s practices regarding user data collected from website activity,
including collection and distribution via third party integrations, like *Google Analytics*.
We take reasonable precautions to protect user data, 
but users are responsible for keeping their own sensitive information secure. 
We may change this policy when necessary.
By using *AntAlmanac*, you consent to this privacy statement.
```
This could be tweaked, but I like this idea

Author: The policy is fundamentally a contract between us (or _AntAlamanac_) and individual users that they agree to as a condition to use the website. It's not just a statement, so we shouldn't use the second person. Also, if we collect sensitive or personally identifiable information, we are responsible; if we don't, then it doesn't concern us, so we can't say that users are categorically responsible for it.

I like the restructuring though—I write excessively long sentences (as demonstrated above). I'll incorporate that.

> <br></br>

Reviewer: ```suggestion

```
I don't think line breaks are needed?

## Information that we collect
**Non-tracking information:**
Information stored on your device that we do not log and that is not used to track usage. This includes, but is not limited to, settings, the last profile loaded on a device, and whether you have dismissed certain popups. This information allows *AntAlmanac* to make your actions and preferences persistent across page accesses.

> <br></br>

Reviewer: ```suggestion

```

**Study plans and usernames:**
Your username(s) and the associated schedules and enrolled classes. This allows you to save your schedules and is NOT shared with third parties.

> <br></br>

Reviewer: ```suggestion

```

**Google Analytics usage data:**
We use *Google Analytics* to track your usage, for example, which button you click on our your search queries. We do so to inform our prioritization and design of features. *Google Analytics* uses cookies and other tracking technologies to collect and store your information. Your usage information is anonymous and aggregated to us. HOWEVER, *Google Analytics* might collect your information individually or track you across websites. To block *Google Analytics* use *Google*’s first-party [browser add-on](https://support.google.com/analytics/answer/181881?hl=en)

> <br></br>

Reviewer: ```suggestion

```

## Information that we do not collect
  **Personally-identifiable information:** We do not collect any personally identifiable information. Your username should not contain your real name, and you should not enter sensitive information into *AntAlmanac*.
  <br></br>
  **Sale of data:** *AntAlamanac* is a not-for-profit project, and we do not sell your information for any purpose.
<br></br>
## Contact us

> If you have questions, email us at: icssc@uci.edu

Reviewer: ```suggestion
If you have questions, email us at: [icssc@uci.edu](mailto:icssc@uci.edu)
```
Email links are fun
